src/eos-updater-fetch.c

```diff
@@ -589,12 +589,11 @@ perform_action_preparation (FlatpakInstallation        *installation,
 }
 
 static gboolean
-pull_flatpaks (GPtrArray     *pending_flatpak_ref_actions,
-               GCancellable  *cancellable,
-               GError       **error)
+pull_flatpaks (FlatpakInstallation  *installation,
+               GPtrArray            *pending_flatpak_ref_actions,
+               GCancellable         *cancellable,
+               GError              **error)
```

Comment:
Rearranging this function to take `FlatpakInstallation` as a parameter could be a separate commit.

Reply:
Maybe ... that first commit would not use the FlatpakInstallation parameter however...